pkg/kapp/clusterapply/cluster_change.go

```diff
@@ -73,10 +76,11 @@ func NewClusterChange(change ctldiff.Change, opts ClusterChangeOpts,
 	identifiedResources ctlres.IdentifiedResources,
 	changeFactory ctldiff.ChangeFactory,
 	changeSetFactory ctldiff.ChangeSetFactory,
-	convergedResFactory ConvergedResourceFactory, ui UI) *ClusterChange {
+	convergedResFactory ConvergedResourceFactory, ui UI,
+	diffMaskRules []ctlconf.DiffMaskRule) *ClusterChange {
```

Comment:
hmm seems a bit weird to provide diffMaskRules into cluster changes... but i guess we need a presentation "concern" for the error so may be thats ok...

Reply:
> hmm seems a bit weird to provide diffMaskRules into cluster changes

I have been wanting to avoid that somehow, but haven't been able to find an alternative till now (maybe there isn't any :( )